sources/packages/backend/libs/integrations/src/esdc-integration/fed-restriction-integration/fed-restriction.processing.service.ts

```diff
@@ -52,9 +52,9 @@ export class FedRestrictionProcessingService {
    */
   async process(processSummary: ProcessSummary): Promise<void> {
     const auditUser = this.systemUsersService.systemUser;
-    // Get the list of all files from SFTP ordered by file name.
+    // Get the list of all ZIP files from SFTP ordered by file name.
     const fileSearch = new RegExp(
-      `^${this.esdcConfig.environmentCode}CSLS\\.PBC\\.RESTR\\.LIST\\.D[\\w]*\\.[\\d]*$`,
+      `^${this.esdcConfig.environmentCode}CSLS\\.PBC\\.RESTR\\.LIST\\.D[\\w]*\\.[\\d]*\\.(zip|ZIP)$`,
```

Comment:
Not enforcing the zip extension would allow the process to download the file either way, compressed or not.
I do not see an AC requesting either one or not and I do not see a reason to have it restricted.
Not a blocker.

Reply:
Not enforcing the `.zip` will also lead to not enforcing the end of file name by removing the existing `$`. I see the benefit of it working either way, but should e remove the `$`. ?

Let me know if I am missing something here.

Reply:
As we talked, IMO, we do need to enforce the precise ending of the string but I am ok either way.